.editorconfig

```diff
@@ -17,9 +17,3 @@ charset = utf-8
 
 # Avoid cookstyle warnings
 trim_trailing_whitespace = true
-
-[*.md]
-indent_size = 4
-
-# Trailing whitespace is meaningful in Markdown
-trim_trailing_whitespace = false
```

Comment:
I’d propose these are things that should be added to the standards, rather than removed, b/c they’re meaningful. For example, trimming trailing whitespace makes it impossible to add a line break in Markdown (the syntax for that is two spaces at the end of a line).

The indent is less of a thing, though the “official” docs (as much as there are any) show 4-space indents for things.

Reply:
I'd recommend you open an issue in https://github.com/sous-chefs/repo-management to address this as this file is managed globally from that repo.